neard/src/migrations.rs

```diff
@@ -6,9 +6,30 @@ use near_chain::{ChainStore, ChainStoreAccess, ChainStoreUpdate, RuntimeAdapter}
 use near_primitives::transaction::ExecutionOutcomeWithIdAndProof;
 use near_primitives::types::{BlockHeight, ShardId};
 use near_store::migrations::set_store_version;
-use near_store::{create_store, DBCol, StoreUpdate};
+use near_store::{create_store, DBCol, Store, StoreUpdate};
 use std::path::Path;
 
+fn delete_col(store: &Store, col: DBCol) -> Result<(), std::io::Error> {
+    let mut store_update = store.store_update();
+    let batch_size_limit = 10_000_000;
+    let mut batch_size = 0;
+    for (key, _) in store.iter_without_rc_logic(col) {
+        store_update.delete(col, &key);
+        batch_size += key.len() + 8;
+        if batch_size > batch_size_limit {
+            store_update.commit()?;
+            store_update = store.store_update();
+            batch_size = 0;
+        }
+    }
+
+    if batch_size > 0 {
+        store_update.commit()?;
+    }
+
+    Ok(())
+}
```

Comment:
migration 8 to 9 uses this to clear a column:
`store.get_rocksdb().unwrap().clear_column(column);`